docs/pages/versions/unversioned/sdk/app-integrity.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: App Integrity
+description: A library that provides access to Google's Play Integrity API on Android and Apple's App Attest service on iOS.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-app-integrity'
+packageName: '@expo/app-integrity'
```

Comment:
should we name it `expo-app-integrity`? we tend to use this convention for expo packages currently. we used `@expo/ui` primarily because npm wouldn't let us register `expo-ui`